packages/button/src/Button/Button.tsx

```diff
@@ -40,7 +42,7 @@ export const Button = React.forwardRef(function Button(
     size,
     darkMode,
     baseFontSize,
-    disabled,
+    disabled: isLoading || disabled,
```

Comment:
let's pull this into variable
`notInteractive = isLoading || disbaled`

Reply:
For future note - I'd be a fan of using explicit states here the way Adam did with Toasts. So we'd have a `state` prop that's either `disabled`, `loading`, or `none`.

Reply:
Definitely +1 for `state = 'disabled' | 'loading' | 'none'`

Reply:
https://jira.mongodb.org/browse/LG-3280

Reply:
It might be easier to have an epic on Jira